.github/workflows/build-images-scheduled.yaml

```diff
@@ -15,3 +15,4 @@ jobs:
       authorinoOperatorVersion: ${{ vars.AUTHORINO_OPERATOR_SHA }}
       limitadorOperatorVersion: ${{ vars.LIMITADOR_OPERATOR_SHA }}
       wasmShimVersion: ${{ vars.WASM_SHIM_SHA }}
+      additionalTags: ${{ github.sha }}
```

Comment:
Actually, this might not even needed, since we are passing the `kuadrantOperatorTag` with the value of `github.sha` in line 13 and 14 of this very same file... so the `build-images-base` already has that tag... does it make sense?

Reply:
Thanks for the suggestion! I totally missed that